Source/WebCore/css/parser/CSSPropertyParserConsumer+Image.cpp

```diff
@@ -693,20 +699,21 @@ static RefPtr<CSSValue> consumeLinearGradient(CSSParserTokenRange& range, const
 
     auto computedColorInterpolationMethod = computeGradientColorInterpolationMethod(colorInterpolationMethod, *stops);
 
-    return CSSLinearGradientValue::create(
-        CSS::LinearGradient {
-            .repeating = repeating,
-            .colorInterpolationMethod = computedColorInterpolationMethod,
-            .gradientLine = gradientLine.value_or(CSS::Vertical { CSS::Bottom { } }),
-            .stops = WTFMove(*stops)
+    return CSSGradientValue::create(
+        CSS::FunctionNotation<Name, CSS::LinearGradient> {
+            .parameters = {
+                .colorInterpolationMethod = computedColorInterpolationMethod,
+                .gradientLine = gradientLine.value_or(CSS::Vertical { CSS::Bottom { } }),
+                .stops = WTFMove(*stops)
+            }
         }
     );
 }
 
 // MARK: <radial-gradient()> | <repeating-radial-gradient()>
 // https://drafts.csswg.org/css-images-4/#radial-gradients
 
-static RefPtr<CSSValue> consumeRadialGradient(CSSParserTokenRange& range, const CSSParserContext& context, CSS::GradientRepeat repeating)
+template<CSSValueID Name> static RefPtr<CSSValue> consumeRadialGradient(CSSParserTokenRange& range, const CSSParserContext& context)
```

Comment:
How did you decide on template argument vs. function argument?

Reply:
Same as above. Needed Name to be a template argument in FunctionNotation.